front_end/panels/ai_chat/core/__tests__/AgentNodesSanitize.test.ts

```
@@ -0,0 +1,80 @@
+        sanitizedSeenInCall = fn;
+        // Return a minimal rawResponse
+        return { rawResponse: { usage: {} } };
+      },
+      parseResponse: (_resp: any) => {
+        // Echo back a tool_call using the same function name seen by provider
+        return { type: 'tool_call', name: sanitizedSeenInCall, args: { q: 1 } };
+      },
+    });
+
+    const state: AgentState = {
+      messages: [ { entity: ChatMessageEntity.USER, text: 'run' } as any ],
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)